### PR TITLE
Loki: Add loki devenv dashboard

### LINKED ISSUE #53046: Devenv: Build Loki devenv dashboard
**Level: medium**

Currently, we don't have any test/sample dashboard for `gdev-loki` data source. We should create a `gdev-dashboard` for loki, similarly as we have for elasticsearch and other data sources - see `devenv/dev-dashboards/datasource-elasticsearch`).

The dashboard could have at least: 
- 2 log panel - (1 log query could have line filter)
- 2 time series panels 
- 2 stats panels 
- template variables 
- ad-hoc filters

### PR DESCRIPTION
**Pull request by itsgareth**
**What this PR does / why we need it**:
this adds a test dashboard for gdev-loki datasource.

**Which issue(s) this PR fixes**:

Fixes #53046 

**Special notes for your reviewer**:
in `/devenv` run `./setup.sh` to load all gdev dashboards
in `/` run `make devenv sources=loki` to load the test data
go to `http://localhost:3000/dashboards` and look for the `Datasource tests - Loki` dashboard

### CODE REVIEW

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+      },
+      "pluginVersion": "9.3.0-pre",
+      "targets": [
+        {
+          "datasource": {
+            "type": "loki",
+            "uid": "PDDA8E780A17E7EF1"
+          },
+          "editorMode": "builder",
+          "expr": "count_over_time({place=\"$place\"} [5s])",
```
Comment (ivanahuckova) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:220]: Could you change this to `5m` which is more realistic.

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+          "legendFormat": "{{place}}",
+          "queryType": "range",
+          "refId": "A"
+        }
+      ],
+      "title": "Luna Log count over 5m",
+      "type": "stat"
+    }
+  ],
+  "refresh": "5s",
```
Comment (ivanahuckova) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:370]: This is too often which results (in my case at least) in `too many outstanding requests`. I'd suggest this to be either `off`, or something like `1m`

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+  "schemaVersion": 37,
+  "style": "dark",
+  "tags": [
+    "gdev",
+    "loki",
+    "datasource-test"
+  ],
+  "templating": {
```
Comment (ivanahuckova) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:378]: Could you also add ad-hoc filters? 🙂

**Review by ivanahuckova — COMMENTED** [review]
😻 😻 😻  

image

This is awesome! Different queries, panels, variables. Great job with this. I've left some feedback bellow. 🙂

**Review by ivanahuckova — APPROVED** [review]
LGTM

**Comment by ivanahuckova** [comment]
Btw I see that some backend test failed in drone, so you need to try to restart the build and/or merge the main. That should resolve it.

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+      },
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
```
Comment (svennergr) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:319]: This line is causing an error. Just remove it and the test should be fine.

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
```
Comment (svennergr) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:256]: This line is causing an error. Just remove it and the test should be fine.

File: devenv/dev-dashboards/datasource-loki/loki_fakedata.json [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
```
Comment (svennergr) [inline devenv/dev-dashboards/datasource-loki/loki_fakedata.json:182]: This line is causing an error. Just remove it and the test should be fine.

**Pull request merged**